src/types/addressPlugin.ts

```diff
@@ -17,4 +18,6 @@ export interface AddressPlugin {
 
   // Not all plugins support scanning:
   scanAddress?: (address: string, checkpoint?: string) => Promise<boolean>
+
+  destroy: () => void
```

Comment:
Can we remove this? Plugins are supposed to be eternal - they are created at boot-up, and live for the lifetime of our process. If you want to clean up your tests, just do `process.exit` to shut down any resources.